tests/shear_heating_with_melt.prm

```diff
@@ -0,0 +1,207 @@
+# Listing of Parameters
+# ---------------------
+# Model for melting at a mid-ocean ridge
```

Comment:
can you add a description what this test does and if it is based on a different test?